package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "the-charity-backend",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "cross-env NODE_ENV=development nest start --watch",
+    "start:debug": "cross-env NODE_ENV=development nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "cross-env NODE_ENV=test jest",
+    "test:watch": "cross-env NODE_ENV=test jest --watch",
+    "test:cov": "cross-env NODE_ENV=test jest --coverage",
+    "test:debug": "cross-env NODE_ENV=test node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "cross-env NODE_ENV=test jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@nestjs/common": "9.0.11",
+    "@nestjs/config": "2.2.0",
+    "@nestjs/core": "9.0.11",
+    "@nestjs/platform-express": "9.0.11",
+    "joi": "17.6.0",
+    "reflect-metadata": "0.1.13",
+    "rimraf": "3.0.2",
+    "rxjs": "7.5.6"
+  },
+  "devDependencies": {
+    "@fullstacksjs/tsconfig": "0.6.0",
+    "@nestjs/cli": "9.1.1",
+    "@nestjs/schematics": "9.0.1",
+    "@nestjs/testing": "9.0.11",
+    "@types/express": "4.17.13",
+    "@types/jest": "29.0.0",
+    "@types/node": "18.7.14",
+    "@types/supertest": "2.0.12",
+    "@typescript-eslint/eslint-plugin": "5.36.1",
+    "@typescript-eslint/parser": "5.36.1",
+    "cross-env": "7.0.3",
+    "eslint": "8.23.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "jest": "29.0.1",
+    "prettier": "2.7.1",
+    "source-map-support": "0.5.21",
+    "supertest": "6.2.4",
+    "ts-jest": "28.0.8",
+    "ts-loader": "9.3.1",
+    "ts-node": "10.9.1",
+    "tsconfig-paths": "4.1.0",
+    "typescript": "4.8.2"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "ts"
+    ],
+    "rootDir": "src",
+    "testRegex": ".*\\.spec\\.ts$",
+    "transform": {
+      ".+\\.(t|j)s$": "ts-jest"
+    },
+    "collectCoverageFrom": [
+      "**/*.(t|j)s"
+    ],
+    "coverageDirectory": "../coverage",
+    "testEnvironment": "node"
+  }
```

Comment:
It's a good practice to keep jest config in a separate folder, as It's possible to cache the install step. Changing package.json invalidates that cache.